### ISSUE TITLE
load_balancer_backend_address_pools on vmss

### ISSUE THREAD

**Issue by CoryDandeneau** [hashicorp/terraform#10349]
Issue with attaching backend address pool from load balancer to vm scale set in azure.

### Terraform Version
0.7.13

### Affected Resource(s)
- azurerm_virtual_machine_scale_set
- load_balancer_backend_address_pool_ids

### Terraform Configuration Files
```hcl
resource "azurerm_virtual_machine_scale_set" "vanessavmss" {
  name = "vanessavmss"
  location = "East US 2"
  resource_group_name = "${azurerm_resource_group.vanessarg.name}"
  upgrade_policy_mode = "Manual"

  sku {
    name = "Standard_A0"
    tier = "Standard"
    capacity = 3
  }

  os_profile {
    computer_name_prefix = "testvm"
    admin_username = "myadmin"
    admin_password = "P@$$word5422"
  }

  os_profile_linux_config {
    disable_password_authentication = false
    }
  network_profile {
      name = "vmssnetwork"
      primary = true
      ip_configuration {
        name = "vmssIPConfig"
        subnet_id = "${azurerm_subnet.vanessainternal.id}"
        load_balancer_backend_address_pool_ids = ["${azurerm_lb_backend_address_pool.vanessabap.id}"]
      }
  }
  storage_profile_os_disk {
    name = "osDiskProfile"
    caching       = "ReadWrite"
    create_option = "FromImage"
    vhd_containers = ["${azurerm_storage_account.vanessasa.primary_blob_endpoint}${azurerm_storage_container.vanessasc.name}"]
  }
  storage_profile_image_reference {
    publisher = "Canonical"
    offer     = "UbuntuServer"
    sku       = "14.04.2-LTS"
    version   = "latest"
  }
}

```
### Expected Behavior
Backend address pool should have been attached to the vmss

### Actual Behavior
The backend address pool is not there so i cannot load balance the scale set

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Try to create VMSS using Terraform and public facing load balancer.

**Comment by vitorarins** [comment]
It might be related to [this TODO](https://github.com/hashicorp/terraform/blob/master/builtin/providers/azurerm/resource_arm_virtual_machine_scale_set.go#L816)?

**Comment by vitorarins** [comment]
Maybe is related to [this TODO](https://github.com/hashicorp/terraform/blob/master/builtin/providers/azurerm/resource_arm_virtual_machine_scale_set.go#L816)?

**Comment by lvdh** [comment]
Terraform v0.8.6 seems to successfully create a VMSS and load_balancer_backend_address_pool_id. It also successfully assigns the load_balancer_backend_address_pool_id to the VMSS.

Can you reproduce with Terraform v0.8.6?

**Comment by vitorarins** [comment]
@lvdh @CoryDandeneau this was fixed with #11516.

**Comment by tombuildsstuff** [comment]
Hi @CoryDandeneau 

As @vitorarins / @lvdh have previously posted - this functionality was added in v0.8.6 / PR #11516 - as such I'm going to close this issue :)

Thanks!
